build.sbt

```diff
@@ -1,22 +1,22 @@
+import sbtorgpolicies.OrgPoliciesPlugin.autoImport.%%
```

Comment:
It shouldn't be needed

Reply:
That import was generated by IntelliJ IDEA. I just removed in the next commit.